pkg/satellite/satellitedb/projectmembers.go

```diff
@@ -0,0 +1,145 @@
+func (pm *projectMembers) GetByMemberID(ctx context.Context, memberID uuid.UUID) (*satellite.ProjectMember, error) {
+	projectMemberDbx, err := pm.db.Get_ProjectMember_By_MemberId(ctx, dbx.ProjectMember_MemberId(memberID[:]))
+	if err != nil {
+		return nil, err
+	}
+
+	return projectMemberFromDBX(projectMemberDbx)
+}
+
+// GetByProjectID is a method for querying project members from the database by projectID.
+func (pm *projectMembers) GetByProjectID(ctx context.Context, projectID uuid.UUID) ([]satellite.ProjectMember, error) {
+	projectMembersDbx, err := pm.db.All_ProjectMember_By_ProjectId(ctx, dbx.ProjectMember_ProjectId(projectID[:]))
+	if err != nil {
+		return nil, err
+	}
+
+	return projectMembersFromDbxSlice(projectMembersDbx)
+}
+
+// Get is a method for querying project member from the database by id.
+func (pm *projectMembers) Get(ctx context.Context, id uuid.UUID) (*satellite.ProjectMember, error) {
+	projectMember, err := pm.db.Get_ProjectMember_By_Id(ctx, dbx.ProjectMember_Id(id[:]))
+	if err != nil {
+		return nil, err
+	}
+
+	return projectMemberFromDBX(projectMember)
+}
+
+// Insert is a method for inserting project member into the database.
+func (pm *projectMembers) Insert(ctx context.Context, memberID, projectID uuid.UUID) (*satellite.ProjectMember, error) {
+	id, err := uuid.New()
+	if err != nil {
+		return nil, err
+	}
+
+	createdProjectMember, err := pm.db.Create_ProjectMember(ctx,
+		dbx.ProjectMember_Id(id[:]),
+		dbx.ProjectMember_MemberId(memberID[:]),
+		dbx.ProjectMember_ProjectId(projectID[:]))
+	if err != nil {
+		return nil, err
+	}
+
+	return projectMemberFromDBX(createdProjectMember)
+}
+
+// Delete is a method for deleting project member by Id from the database.
+func (pm *projectMembers) Delete(ctx context.Context, id uuid.UUID) error {
+	_, err := pm.db.Delete_ProjectMember_By_Id(ctx, dbx.ProjectMember_Id(id[:]))
+
+	return err
+}
+
+// Update is a method for updating project member entity.
+func (pm *projectMembers) Update(ctx context.Context, projectMember *satellite.ProjectMember) error {
+	_, err := pm.db.Update_ProjectMember_By_Id(ctx,
+		dbx.ProjectMember_Id(projectMember.ID[:]),
+		dbx.ProjectMember_Update_Fields{
+			ProjectId: dbx.ProjectMember_ProjectId(projectMember.ProjectID[:]),
+		})
+
+	return err
+}
+
+// projectMemberFromDBX is used for creating ProjectMember entity from autogenerated dbx.ProjectMember struct
+func projectMemberFromDBX(projectMember *dbx.ProjectMember) (*satellite.ProjectMember, error) {
+	if projectMember == nil {
+		return nil, errs.New("projectMember parameter is nil")
+	}
+
+	id, err := bytesToUUID(projectMember.Id)
+	if err != nil {
+		return nil, err
+	}
+
+	memberID, err := bytesToUUID(projectMember.MemberId)
+	if err != nil {
+		return nil, err
+	}
+
+	projectID, err := bytesToUUID(projectMember.ProjectId)
+	if err != nil {
+		return nil, err
+	}
+
+	u := &satellite.ProjectMember{
```

Comment:
nit, `return &satellite.ProjectMember{ ... }, nil`, makes it slightly more clear that this is the final result.